src/main/java/com/akukhtin/springbootapplication/service/impl/UserServiceImpl.java

```diff
@@ -12,8 +14,13 @@
     private UserRepository userRepository;
 
     @Override
-    public User update(User user) {
-        return userRepository.save(user);
+    public Optional<User> update(Long id, UserDto userDto) {
+        User newUser = userRepository.findById(id).orElseThrow();
```

Comment:
```suggestion
        User newUser = userRepository.findById(id).orElseThrow(() -> your exception);
```
@AlexKukhtinDev ми проговорювали що писати orElseThrow() не треба